File: RxTools-library/src/main/java/com/vondear/rxtools/view/dialog/dialogWheel/OnWheelClickedListener.java
package com.vondear.rxtools.view.dialog.dialogWheel;

/**
 * Wheel clicked listener interface.
 * <p>The onItemClicked() method is called whenever a wheel item is clicked
 * <li> New Wheel position is set
 * <li> Wheel view is scrolled
 */
public interface OnWheelClickedListener {
    /**
     * Callback method to be invoked when current item clicked
     * @param wheel the wheel view
     * @param itemIndex the index of clicked item
     */
    void onItemClicked(WheelView wheel, int itemIndex);
}

File: RxUI/src/main/java/com/tamsiree/rxui/view/dialog/wheel/OnWheelClickedListener.kt
package com.tamsiree.rxui.view.dialog.wheel

/**
 * @author tamsiree
 * Wheel clicked listener interface.
 *
 * The onItemClicked() method is called whenever a wheel item is clicked
 *  *  New Wheel position is set
 *  *  Wheel view is scrolled
 */
interface OnWheelClickedListener {
    /**
     * Callback method to be invoked when current item clicked
     * @param wheel the wheel view
     * @param itemIndex the index of clicked item
     */
    fun onItemClicked(wheel: WheelView?, itemIndex: Int)
}